doc/source/changes/version_0_28.rst.inc

```diff
@@ -283,6 +283,9 @@ Miscellaneous improvements
 
   Closes :issue:`549`.
 
+* Renamed argument `sheet_name` to `sheet` on array.py `to_excel` def.
+  Rename `sheet` initial occurences to `sheetobj` to prevent from rename errors.
```

Comment:
Please remove this line. The changelog is for users so they do not care about such details.